molecule_generation/utils/moler_inference_server.py

```diff
@@ -11,6 +11,8 @@
 from rdkit import Chem
 from more_itertools import chunked, ichunked
 
+from molecule_generation.models.moler_generator import MoLeRGenerator
+from molecule_generation.utils.moler_decoding_utils import DecoderSamplingMode
 from molecule_generation.dataset.in_memory_trace_dataset import InMemoryTraceDataset, DataFold
 from molecule_generation.models.moler_vae import MoLeRVae
 from molecule_generation.utils.moler_decoding_utils import MoLeRDecoderState
```

Comment:
nit: sort and combine imports
```suggestion
from molecule_generation.dataset.in_memory_trace_dataset import InMemoryTraceDataset, DataFold
from molecule_generation.models.moler_generator import MoLeRGenerator
from molecule_generation.models.moler_vae import MoLeRVae
from molecule_generation.utils.moler_decoding_utils import DecoderSamplingMode, MoLeRDecoderState
```